components/button/ButtonSocialMedia.tsx

```diff
@@ -0,0 +1,25 @@
+import clsx from 'clsx'
+import { Image, ImageSourcePropType, Pressable, Text } from 'react-native'
+
+interface Props {
+  image: ImageSourcePropType
+  label?: string | undefined
+}
+
+export default function ButtonSocialMedia({ image, label }: Props) {
```

Comment:
evitar exportar componente de manera directa en la declaración, en su lugar : 

```suggestion
function ButtonSocialMedia({ image, label }: Props) {}

export default ButtonSocialMedia
```